cmd/zpool/zpool_main.c

```diff
@@ -1923,6 +2041,49 @@ print_status_config(zpool_handle_t *zhp, status_cbdata_t *cb, const char *name,
 		}
 	}
 
+	/* Optionally display vdev initialization status for leaves */
+	if (cb->cb_print_vdev_init && children == 0) {
```

Comment:
What specifically is the usability concern around displaying the init status by default?

When an initialization is in progress, how would admin's typically discover this?  I'm concerned that if they have to know to add the `-i` flag, they may miss this and not realize why their pool's performance is different during initialization.  We should consider printing the status by default if it is `VDEV_INITIALIZE_ACTIVE`.

Reply:
My specific concern is readability.  For anyone using a standard 80 column terminal the `pool status` output becomes needlessly challenging to read for the lifetime of the pool.

Though I can see the argument for wanting to see this while in the `VDEV_INITIALIZE_ACTIVE`state.  Why don't we print `(initializing)` when active, which is concise and consistent with the long standing behavior of printing `(resilvering)` or `(repairing)` after a vdev.  The new `-i` can then be used to report the detailed status.

Reply:
That sounds good to me.